src/main/java/core/basesyntax/controller/AllCarsController.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.lib.Injector;
+import core.basesyntax.model.Car;
+import core.basesyntax.service.CarService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AllCarsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("core.basesyntax");
```

Comment:
```suggestion
    private static final Injector INJECTOR= Injector.getInstance("core.basesyntax");
```
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html#:~:text=Class%20names%20should%20be%20nouns,such%20as%20URL%20or%20HTML

Reply:
Fixed